gulpfile.js

```diff
@@ -76,39 +23,58 @@ gulp.task('clean', function () {
   ]);
 });
 
-gulp.task('pull:candidates', function () {
-  return gulp.src(dataDir('candidate', '*', 'index.json'))
-    .pipe(slugifyName((data) => `${slugify(data.name)}.json`))
+gulp.task('pull:ballots', function () {
+  return gulp.src(dataDir('_ballots', '**', '*.md'))
+    .pipe(gulp.dest('_ballots'));
+});
+
+gulp.task('pull:referendums', function () {
+  return gulp.src(dataDir('_referendums', '**', '*.md'))
+    .pipe(gulp.dest('_referendums'));
+});
+
+gulp.task('pull:referendums', function () {
+  return gulp.src(dataDir('_referendums', '**', '*.md'))
+    .pipe(gulp.dest('_referendums'));
+});
+
+gulp.task('pull:candidates_data', function () {
+  return gulp.src(dataDir('_data', 'candidates', '**', '*.json'))
     .pipe(gulp.dest('_data/candidates'));
 });
 
-gulp.task('pull:contributions', function () {
-  return gulp.src(dataDir('committee', '*', 'contributions', 'index.json'))
-    .pipe(contributions())
-    .pipe(gulp.dest('_data/contributions'));
+gulp.task('pull:candidates', function () {
+  return gulp.src(dataDir('_candidates', '**', '*.md'))
+    .pipe(gulp.dest('_candidates'));
+});
+
+gulp.task('pull:committees', function () {
+  return gulp.src(dataDir('_data', 'committees', '*.json'))
+    .pipe(gulp.dest('_data/committees'));
+});
+
+gulp.task('pull:office_elections', function () {
+  return gulp.src(dataDir('_office_elections', '**', '*.md'))
+    .pipe(gulp.dest('_office_elections'));
 });
 
 gulp.task('pull:referendum_opposing', function () {
-  return gulp.src(dataDir('referendum', '*', 'opposing', 'index.json'))
-    .pipe(slugifyName((data) => {
-      const election = guessElection(data);
-      return `oakland/${election}/${slugify(data.title)}.json`
-    }))
+  return gulp.src(dataDir('_data', 'referendum_opposing', '**', '*.json'))
     .pipe(gulp.dest('_data/referendum_opposing'));
 });
 
 gulp.task('pull:referendum_supporting', function () {
-  return gulp.src(dataDir('referendum', '*', 'supporting', 'index.json'))
-    .pipe(slugifyName((data) => {
-      const election = guessElection(data);
-      return `oakland/${election}/${slugify(data.title)}.json`
-    }))
+  return gulp.src(dataDir('_data', 'referendum_supporting', '**', '*.json'))
     .pipe(gulp.dest('_data/referendum_supporting'));
 });
 
 gulp.task('pull', gulp.parallel(
+  'pull:ballots',
   'pull:candidates',
-  'pull:contributions',
```

Comment:
Hmm, yep, we deleted the contributions task. Are the contributions supposed to be included in the committee finance data?